standards.md

```diff
@@ -47,7 +47,7 @@ Documentation is needed for DEM dependencies on specific Python module or versio
 
 The completed DEM should be registered with one or more Digital Object Identifiers (DOIs). This also applies to PODs that are merged into the MDTF framework. A service such as Zenodo can be used to obtain DOIs for software. 
```

Comment:
Explicitly stating who does this, would help. I imagine its the user.

Reply:
I've adding wording saying that the developer should register the DEM.